extensions/infobip/v1/actions/sendSms/config/fields.ts

```diff
@@ -0,0 +1,42 @@
+import { z, type ZodTypeAny } from 'zod'
+import {
+  type Field,
+  FieldType,
+  StringType,
+  makeStringOptional,
+} from '@awell-health/extensions-core'
+
+export const fields = {
+  from: {
+    label: 'From',
+    id: 'from',
+    type: FieldType.STRING,
+    stringType: StringType.PHONE,
+    required: false,
+    description:
+      'The phone number you wish to use for sending the text message. Defaults to value provided in settings.',
+  },
+  to: {
+    label: 'To',
+    id: 'to',
+    type: FieldType.STRING,
+    stringType: StringType.PHONE,
+    required: true,
+    description:
+      'The phone number to which you intend to send the text message',
```

Comment:
Specify formatting of static or data point value (E164)

Reply:
In the light of "Keeping things simple", I don't see any value in communicating the format here.

- Most users, especially non-tech ones will not know what "E.164" is
- We have a `telephone` data point type that can only store E.164 numbers so validation is already happening somewhere else in the product. I.e. if a user selects a telephone data point, we know for a fact it's correctly formatted
- If a user enters a static value (highly unlikely), we render an input field that forces them to enter the number in E.164 format.